lib/ssm_config/migration_helper.rb

```diff
@@ -51,11 +51,17 @@ def accessor_key_array(arr, curr = '')
 
     def determine_class(value)
       return 'boolean' if (value == false) || (value == true)
-      return value.class
+      return 'erb' if (value[0..2] == '<%=') && (value[-2..-1] == '%>')
```

Comment:
I think this logic might fail determing this one because of double quotes
https://github.com/bodyshopbidsdotcom/snapsheet-tx/blob/6a9c286de2ec90f7ea9eb3b5cb373dd82e880665/config/smart_payables.yml#L22-L23

Reply:
YML being read in handles that already and all you get here is just the `<%= ... %>` string.